docs/api/menu-item.md

```diff
@@ -122,6 +122,46 @@ Lowercase `role`, e.g. `toggledevtools`, is still supported.
 
 The following properties are available on instances of `MenuItem`:
 
+#### `menuItem.id`
+
+A `String` indicating the item's unique id, this property can be
+dynamically changed.
+
+#### `menuItem.label`
+
+A `String` indicating the item's visible label, this property can be
+dynamically changed.
+
+#### `menuItem.click`
+
+A `Function` that is fired when the MenuItem receives a click event.
+
+#### `menuItem.submenu`
+
+A `MenuItemConstructorOptions[] | Menu` (optional) containing the menu
+item's submenu, if present.
+
+#### `menuItem.type`
+
+A `String` indicating the type of the item.
+
+#### `menuItem.role`
+
+A `String` (optional) indicating the item's role, if set.
+
+#### `menuItem.accelerator`
+
+A `String` (optional) indicating the item's accelerator, if set.
+
+#### `menuItem.icon`
+
+A `[NativeImage](native-image.md) | String` (optional) indicating the
```

Comment:
We do not support linking `native-image` in this part of the docs-parser.  I'd suggest changing this to `NativeImage | String`

Reply:
I changed it to `NativeImage | String` locally but it only comes out as `NativeImage` in the definitions.

Reply:
@nixxquality I guess the docs parser hasn't been made to handle the multi-type case for properties then 🤔

Reply:
If the docs are correct with `NativeImage | String`, maybe let's open a separate issue for the parser, but not block on it here?

Reply:
Sounds good to me, let's ship correct docs and fix the generator async